### PR TITLE
Hotfix/t/web 2012 new detail transactions friendly name bug

### PR DESCRIPTION
**Pull request by Vicentecarrillo**
resurrection of https://github.com/Invoca/backdraft/pull/56

### CODE REVIEW

**Comment by Vicentecarrillo** [comment]
@albertovillalobos @nburwell @mikeweaver @codyjca

**Comment by albertovillalobos** [comment]
@nburwell @mikeweaver I've addressed your CR comments.

**Comment by Vicentecarrillo** [comment]
merging this in, please address any further CR on 96/t/integration_branch

File: src/plugins/data_table/column_types/attr.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ app.view.dataTable.columnType(function(columnType) {
   });
 
   columnType.nodeMatcher(function(config) {
-    return "." + Backdraft.Utils.toCSSClass(config.title);
+    return "." + Backdraft.Utils.toCSSClass(config.attr || config.title);
```
Comment (nburwell) [inline src/plugins/data_table/column_types/attr.js]: this line was missed (or is it needed for some other code path?)

File: src/plugins/data_table/data_table.js [hunk 2]
```diff
@@ -431,7 +431,19 @@ var LocalDataTable = (function() {
         // here we use the text in the header to get the column config by title
         // there isn't a better way to do this currently
         var title = this.textContent || this.innerText;
-        var col = cg.columnConfigByTitle.attributes[title];
+        var col;
+        // Try to grab it by attribute, if possible
+        var matches = this.className.match(/column-([^\s]*)/);
+        if (matches && matches[1]) {
+          cg.columnsConfig.forEach(function(currentObject){
```
Comment (nburwell) [inline src/plugins/data_table/data_table.js]: a better name here is: `currentObject` -> `currentColConfig`

File: src/utils/css.js [hunk 3]
```diff
@@ -2,6 +2,6 @@
 Backdraft.Utils.toCSSClass = (function() {
   var cssClass = /[^a-zA-Z_0-9\-]/g;
   return function(input) {
-    return input.replace(cssClass, "-");
+    return "column-"+input.replace(cssClass, "-");
```
Comment (nburwell) [inline src/utils/css.js]: Something missed from our discussions that would be great to throw in.. `column-` appears now 3 times in the repo, and isn't dry (or prone to updating without causing bugs).

How about:

```
Backdraft.Utils.CSS_CLASS_PREFIX = "column-";
```

**Pull request merged**